src/runtime/contrib/cudnn/cudnn_utils.h

```diff
@@ -54,6 +55,15 @@ inline void GetStride(int nbdim, const int *dims, int *strides) {
   }
 }
 
+inline void GetCudnnStride(int nbdim,
+                           const std::vector<int>& dims,
+                           std::vector<int>* strides) {
```

Comment:
std::vector<int>& strides

Reply:
If using reference, it would cause lint failure: "Is this a non-const reference? If so, make const or use a pointer: std::vector<int>& strides  [runtime/references] [2]"

Reply:
ok, didn't know the linter favors such style.